### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by milkorlife** [nikolakis1919/RokTracker#22]
Hey this is more of an  half request. Some accounts in lost kingdom can get locked out there and won't be able to be viewed by tool or manually but their power will still be in top 300. Would be really sweet if could add them just power instead of hopping over those.

Also any chance for formula for Excel to calculate t4/t5 gains , deads.

**Comment by nikolakis1919** [comment]
Hello!

Excel calculation formula will be a feature to consider. About the request you made,  I think it's not how the game works. If your city cannot be located on the map, it's not measured in the total power(stats) calculation of each kingdom. If we can verify the opposite via support, I will consider adding that, but for now, it's not under consideration.

Thank you for your support. I will keep this open as an improvement.

**Comment by milkorlife** [comment]
Yeah i found fork of your tool that does look at those accs so i guess it's optional. Thanks for this tool !
